graphql_compiler/compiler/ir_lowering_sql/__init__.py

```diff
@@ -223,3 +231,20 @@ def _map_block_index_to_location(ir_blocks):
                 block_index_to_location[ix] = ir_block.location
             current_block_ixs = []
     return block_index_to_location
+
+
+def lower_unary_transformations(ir_blocks):
+    """Raise exception if any unary transformation block encountered."""
+    def visitor_fn(expression):
+        """Raise error if current expression is a UnaryTransformation."""
+        if not isinstance(expression, expressions.UnaryTransformation):
+            return expression
+        raise NotImplementedError(
+            u'UnaryTransformation expression "{}" is unsupported by SQL backend.'.format(expression)
```

Comment:
nit: if you include the `ir_blocks` in the message here, it will be easier to figure out where the `UnaryTransformation` emerged from, and why